tools/launcher/common/hf_ptq/hf_ptq.sh

```
@@ -0,0 +1,39 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+SCRIPT_DIR="$(dirname "$(readlink -f "$0")")"
+source ${SCRIPT_DIR}/../service_utils.sh
+
+trap 'error_handler $0 $LINENO' ERR # ERROR HANDLER
+trap 'exit_handler' EXIT
+###################################################################################################
+
+cd modules/Model-Optimizer/examples/llm_ptq
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)